javascript/chronometer.js

```diff
@@ -1,15 +1,19 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = 0;
   }
   startClick(callback) {
-    // ... your code goes here
+    setInterval(
+      () => this.currentTime++, 1000)
```

Comment:
Great stuff!

Two matters could be improved:
1. Perhaps declare the function outside the setInterval method and submit it as the first argument of the setInterval method.
2. The callback should be (conditionally) called.